src/elona/lua_env/lua_api/lua_api.cpp

```diff
@@ -1,28 +1,53 @@
 #include "lua_api.hpp"
 
+#include "lua_api_animation.cpp"
+#include "lua_api_chara.cpp"
+#include "lua_api_debug.cpp"
+#include "lua_api_fov.cpp"
+#include "lua_api_gui.cpp"
+#include "lua_api_i18n.cpp"
+#include "lua_api_input.cpp"
+#include "lua_api_item.cpp"
+#include "lua_api_magic.cpp"
+#include "lua_api_map.cpp"
+#include "lua_api_math.cpp"
+#include "lua_api_pos.cpp"
+#include "lua_api_rand.cpp"
+#include "lua_api_skill.cpp"
+#include "lua_api_trait.cpp"
+#include "lua_api_world.cpp"
+
 namespace elona
 {
 namespace lua
 {
 
+#define LUA_API_BIND(core, name) \
+    { \
+        sol::table table = core.create_named(#name); \
+        name::bind(table); \
+    }
+
```

Comment:
The field name created by `create_named` has `LuaApi` prefix. The following macro works well instead:

```cpp
#define LUA_API_BIND(core, name) \
    { \
        sol::table table = core.create_named(#name); \
        LuaApi##name::bind(table); \
    }
```

... and pass non-prefix name like `Chara`.